cli/dcoscli/data/help/task.txt

```diff
@@ -4,11 +4,15 @@ Description:
 Usage:
     dcos task --help
     dcos task --info
+    dcos task exec [--interactive --tty] <task> <cmd> [<args>...]
     dcos task log [--completed --follow --lines=N] [<task>] [<file>]
     dcos task ls [--long --completed] [<task>] [<path>]
     dcos task [--completed --json <task>]
 
 Command:
+    exec
+        Launch a process inside a container with the given <task_id>
```

Comment:
Suggest: `Launch a process (`<cmd>`) inside of a container for a task (`<task_id>`).